build.gradle

```diff
@@ -157,12 +157,12 @@ group = "org.grails"
 // directories created during the build which are related
 // to turning the workspace root into a GRAILS_HOME
 ext {
-    distInstallDir = new File(project.buildDir, "dist-tmp")
-    homeDistDir = new File(project.buildDir, "install/dist")
-    homeBinDir = new File(project.buildDir, "install/bin")
-    homeConfDir = new File(project.buildDir, "install/conf")
-    homeLibDir = new File(project.buildDir, "install/lib")
-    homeSrcDir = new File(project.buildDir, "install/src")
+    distInstallDir = project.getLayout().getBuildDirectory().dir("dist-tmp").get()
```

Comment:
I think `layout.buildDirectory` would work instead of `project.getLayout().getBuildDirectory()`??

Reply:
I made this adjustment for each change.